test_cases/stdlib/check_sqlite3.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import sqlite3
+from contextlib import closing
+from typing_extensions import assert_type
+
+# https://github.com/python/typeshed/issues/11558
```

Comment:
This comment is now obsolete.

Reply:
Comment removed: shawnbrown/typeshed@9a13166